core/modules/file/file.module

```diff
@@ -429,6 +429,15 @@ function file_menu() {
     'file' => 'file.pages.inc',
   );
 
+  $items['admin/reports/status/classify'] = array(
```

Comment:
The closest example I can think of to this kind of batch update is the UTF8-MB4 database update, which lives at `admin/config/development/utf8mb4-upgrade`, basically in the menu where it would be if it were a normal menu item.

Since this is file-type related, I'd probably put it at:
```suggestion
  $items['admin/structure/file-types/classify'] = array(
```

Reply:
Done